packages/data/addon/adapters/facts/elide.ts

```diff
@@ -71,12 +63,33 @@ export default class ElideFactsAdapter extends EmberObject implements NaviFactAd
     return moment(value).format(this.grainFormats[grain]);
   }
 
+  private filterBuilders: Record<FilterOperator, (field: string, value: string[]) => string> = {
+    eq: (f, v) => `${f}==('${v[0]}')`,
+    neq: (f, v) => `${f}!=('${v[0]}')`,
+    in: (f, v) => `${f}=in=(${v.map(e => `'${e}'`).join(',')})`,
+    notin: (f, v) => `${f}=out=(${v.map(e => `'${e}'`).join(',')})`,
+    contains: (f, v) => `${f}=in=(${v.map(e => `'*${e}*'`).join(',')})`,
+    null: (f, _v) => `${f}=isnull=true`,
+    notnull: (f, _v) => `${f}=isnull=false`,
```

Comment:
I think we should either adopt the single `isnull` type operator within navi with `[true]` or `[false]` as values or allow for values to be an empty array.

Reply:
Agreed. We will need to change a few things for that. Let's come back.